examples/data-block-examples/esa-sentinel5p-l3.ipynb

```diff
@@ -0,0 +1,347 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "introductory-baker",
+   "metadata": {},
+   "source": [
+    "# ESA Sentinel-5P L3 (GeoTIFF)\n",
+    "Sentinel-5P L3 (GeoTIFF) provides clipped Level 3 (daily composites) as Cloud Optimized GeoTIFFs (COGs) from Sentinel-5P mission. Sentinel-5P is a polar orbiting satellite that completes 14 orbits of the Earth a day. It carries the TROPOspheric Monitoring Instrument (TROPOMI) which is a spectrometer that senses ultraviolet (UV), visible (VIS), near (NIR) and short wave infrared (SWIR) to monitor ozone, methane, formaldehyde, aerosol, carbon monoxide, nitrogen dioxide and sulphur dioxide in the atmosphere.\n",
+    "\n",
+    "For more information, refer to [UP42 Documentation](https://docs.up42.com/up42-blocks/data/esa-s5p-l3-gtiff.html) and [UP42 Marketplace](https://marketplace.up42.com/block/70919851-c563-4e5e-9393-029cf10377c7)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "racial-teach",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import geojson\n",
+    "import up42\n",
+    "import geopandas as gpd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "strange-watch",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2021-03-11 15:56:50,467 - Authentication with UP42 successful!\n",
+      "2021-03-11 15:56:51,226 - Initialized Project(name: Blocks, project_id: 810e8039-cb5c-445b-b08d-53b0dde1bde0, description: , createdAt: 2020-07-13T14:32:49.406Z)\n",
+      "2021-03-11 15:56:51,226 - Getting existing workflows in project ...\n",
+      "2021-03-11 15:56:51,657 - Got 54 workflows for project 810e8039-cb5c-445b-b08d-53b0dde1bde0.\n",
+      "100%|██████████| 54/54 [00:21<00:00,  2.51it/s]\n",
+      "2021-03-11 15:57:13,220 - Using existing workflow: esa-s5p-workflow - 1ecf3424-4dae-44a2-ac6b-79311dbc8089\n"
+     ]
+    }
+   ],
+   "source": [
+    "up42.authenticate(project_id=\"12345\", project_api_key=\"12345\")\n",
+    "project = up42.initialize_project()\n",
+    "\n",
+    "# Construct workflow\n",
+    "workflow = project.create_workflow(name=\"esa-s5p-workflow\", use_existing=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "floating-software",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2021-03-11 15:57:16,291 - Added tasks to workflow: [{'name': 'esa-s5p-l3-gtiff:1', 'parentName': None, 'blockId': '70919851-c563-4e5e-9393-029cf10377c7'}]\n",
+      "2021-03-11 15:57:16,291 - Getting small example aoi in location 'Berlin'.\n"
+     ]
+    }
+   ],
+   "source": [
+    "input_tasks = [\"esa-s5p-l3-gtiff\"]\n",
+    "workflow.add_workflow_tasks(input_tasks)\n",
+    "# Define the aoi \n",
+    "aoi = up42.get_example_aoi(as_dataframe=False)\n",
```

Comment:
I guess you don't need this since you specify your own aoi in next cell, right?

Reply:
Good catch!